lib/ruby/stdlib/nkf.rb

```diff
@@ -0,0 +1,2 @@
+# Load built-in nkf library
+org.jruby.ext.nkf.RubyNKF.createNKF(JRuby.runtime)
```

Comment:
you could add a static `RubyNKF.load(org.jruby.Ruby)` which delegates to `createNKF`
its not loaded on boot and (guessing NKF) is rarely used thus not really  blocker